metadata-ingestion/src/datahub/cli/cli_utils.py

```diff
@@ -135,7 +139,24 @@ def get_details_from_config():
 
 
 def get_details_from_env() -> Tuple[Optional[str], Optional[str]]:
```

Comment:
Shouldn't we check if `DATAHUB_GMS_HOST` environment variable is in the old or the in the new format?
This will be a non backward compatible change for those users who has `DATAHUB_GMS_HOST` set to for example `https://myhost:8443`
I think we should check it's format and if it has protocol and port then we should set automatically the `ENV_METADATA_HOST_URL` property and also write some deprecation warning.